test/publish/basic_test.rb

```diff
@@ -1,28 +1,33 @@
 require 'browser_helper'
 
-describe 'publish-basic', :js do
+describe :publish, :js do
+  describe :basic do
+    it 'gets redirected to the login page' do
+      visit CONFIG.PUBLISH_HOST
+      page.must_have_content "You must login"
+    end
 
-  it 'gets redirected to the login page' do
-    visit CONFIG.PUBLISH_HOST
-    page.must_have_content "You must login"
+    it 'can login' do
+      publish_login!
+      visit CONFIG.PUBLISH_HOST
+      page.must_have_content 'Home'
+      page.must_have_content 'Your Series'
+      page.must_have_content 'Your Standalone Episodes'
+    end
   end
 
-  it 'can login' do
-    publish_login!
-    visit CONFIG.PUBLISH_HOST
-    page.must_have_content 'Home'
-    page.must_have_content 'Your Series'
-    page.must_have_content 'Your Standalone Episodes'
-  end
-
-  # TODO: Intl errors prevent waiting for spinners to disappear
-  # it 'can navigate to a story' do
-  #   publish_visit CONFIG.PUBLISH_HOST
-  #   all('publish-home-story')[1].click
-  #   publish_wait
-  #   page.must_have_content 'Edit Episode'
-  #   page.must_have_content 'Episode Title'
-  #   page.must_have_content 'Cover Image'
-  # end
+  describe :audio do
+    after do
+      puts "Cleaning up test series and episodes" if debug?
+      delete_test_series_and_episodes!
```

Comment:
So it seems like the audio tests take ~60s to run, most of which is this cleanup.  Is that what you're seeing?

I'm hopeful that we can speed that up substantially with upcoming CMS changes.  I suspect it's the dashboard load and "search for series" that's slowing it down.

Reply:
yes, the page load times are the biggest bottleneck, agreed.
and before I had the auto-clean up stuff working, I had 100+ series/episodes to clean up!